srcpkgs/cross-x86_64-w64-mingw32/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'cross-x86_64-w64-mingw32'
-_gcc_version="8.3.0"
-_binutils_version="2.32"
-_gmp_version="6.1.2"
-_mpfr_version="4.0.1"
-_mpc_version="1.1.0"
-_isl_version="0.19"
-_mingw_version="6.0.0"
+_gcc_version=9.1.0
+_binutils_version=2.32
+_gmp_version=6.1.2
+_mpfr_version=4.0.1
+_mpc_version=1.1.0
+_isl_version=0.19
+_mingw_version=6.0.0
 
 pkgname=cross-x86_64-w64-mingw32
-version="$_mingw_version"
-revision=1
-archs="x86_64* i686*"
+version=$_mingw_version
+revision=2
+archs="x86_64 i686"
```

Comment:
why is this restricted to glibc again?

Reply:
Seems to be due to me doing a wrong upstream merge at some point.

Reply:
fixed